app/src/components/Home/CoverPage.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import $ from 'jquery';
 import CoverPageStyle from '../../../styles/components/c-cover-page.scss';
-import Slider from 'react-slick';
+import Slider from '../../lib/react-slick.min';
```

Comment:
@tiagojsag The version deployed on `npm` hasn't been updated since March and contains an issue that has been fixed on `master`.